consensus/istanbul/backend/engine.go

```diff
@@ -692,8 +692,10 @@ func (sb *backend) snapshot(chain consensus.ChainReader, number uint64, hash com
 		}
 		// No snapshot for this header, gather the header and move backward
 		if header := getPrevHeaderAndUpdateParents(chain, number, hash, &parents); header == nil {
+			logger.Info("header null")
 			return nil, consensus.ErrUnknownAncestor
 		} else {
+			logger.Info("header not null", "header", header)
```

Comment:
If we are going to leave this log, printing all `header` information is not necessary.

Reply:
Please remove unnecessary logging